doc/languages-frameworks/beam.section.md

```diff
@@ -52,15 +54,151 @@ Erlang.mk functions similarly to Rebar3, except we use `buildErlangMk` instead o
 
 #### Mix Packages {#mix-packages}
 
-Mix functions similarly to Rebar3, except we use `buildMix` instead of `buildRebar3`.
+`mixRelease` is used to make a release in the mix sense. Dependencies will need to be fetched with `fetchMixDeps` and passed to it.
+
+#### mixRelease - Elixir Phoenix example
+
+Here is how your `default.nix` file would look.
+
+```nix
+with import <nixpkgs> { };
```

Comment:
That wouldn't work on nixos where nixpkgs can be called nixos.

Reply:
I was trying to take the most common case here.
What do you propose?
Should I add a comment like
`Replace nixpkgs with nixos depending on how your main channel is called`

Reply:
> Replace nixpkgs with nixos if you are on NixOS. If your default channel is called something different you should insert its name instead.

Something like the above I think should be okay.

Reply:
Just added it

Reply:
Just added it

Reply:
Wait, what? No. `nixpkgs` Nix path entry is always called `nixpkgs`, regardless of the channel name.

Reply:
Thanks for the comment. Today I learned something. Let me remove this.

Reply:
Ok should be good. Feel free to ping me, if you think it needs something else.